SYCL/AtomicRef/device_has_aspect_atomic64.cpp

```diff
@@ -0,0 +1,63 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out %opencl_lib
+// %level_zero_options RUN: %CPU_RUN_PLACEHOLDER %t.out RUN:
+// %GPU_RUN_PLACEHOLDER %t.out RUN: %ACC_RUN_PLACEHOLDER %t.out
+
+// REQUIRES: (opencl && opencl_icd) || (level_zero && level_zero_dev_kit) || hip
```

Comment:
Why is the CUDA backend excluded from this?

Reply:
This test is intended for modified backend's implementations of device.has(aspect::atomic64) in pull/5430 (described above), and since the implementation on CUDA has not changed, this test does not run on it.

Reply:
* HIP is included due to removal of the 'get_device_info<bool, info::device::atomic64>' safe specialization, causing some HIP tests to fail due to lack of aspect::atomic64 check implementation.

Reply:
Okay. CUDA should support the aspect and will return `true` or `false` based on the SM version. I don't know if there is a test for it, but I think it may make sense to split it into another test where we can check with different SM versions.

However, could we have a case for CUDA here where it just ensures that it doesn't crash when getting whether or not it has the aspect?

Reply:
Okay, I'll add a CUDA case that just checks working of device.has(aspect::atomic64).